spacelift/data_azure_integrations_test.go

```diff
@@ -0,0 +1,42 @@
+package spacelift
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	. "github.com/spacelift-io/terraform-provider-spacelift/spacelift/internal/testhelpers"
+)
+
+func TestAzureIntegrationsData(t *testing.T) {
+	t.Run("when looking up integrations", func(t *testing.T) {
+		randomID := acctest.RandStringFromCharSet(5, acctest.CharSetAlphaNum)
+
+		resourceName := "spacelift_azure_integration.test"
+		datasourceName := "data.spacelift_azure_integrations.test"
+
+		testSteps(t, []resource.TestStep{{
+			Config: fmt.Sprintf(`
+			resource "spacelift_azure_integration" "test" {
```

Comment:
I think it's worth creating more than one integration here since the data source returns multiple.